tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -850,4 +852,81 @@ public void submit() {
     }
   }
 
+  /**
+   * Object used to keep some data when a configuration change happens and the activity is
+   * re-created.
+   * It's boiler-plate but this is how to save View state.
+   */
+  private static class SavedState extends BaseSavedState {
+    /* This will store the current scale and position */
+    float scale;
```

Comment:
yeah package-private is `m` https://source.android.com/source/code-style.html - i guess you can just reference them directly in the same package although that looks a little weird, or add getters, up to you